changelog.d/7189.change

```diff
@@ -0,0 +1 @@
+Voice Broadcast: Inform the user about decryption errors during a voice broadcast.
```

Comment:
You could remove `Voice Broadcast:`the description is explicit IMO.

Reply:
done!